opentracing-shim/src/opentracingshim/__init__.py

```diff
@@ -0,0 +1,254 @@
+# Copyright 2019, OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from contextlib import contextmanager
+
+import opentracing
+
+from opentelemetry.trace import Event, Span, SpanContext, Tracer
+from opentracingshim import util
+
+
+def create_tracer(tracer: Tracer) -> opentracing.Tracer:
+    return TracerWrapper(tracer)
+
+
+class SpanContextWrapper(opentracing.SpanContext):
+    def __init__(self, otel_context: SpanContext):
+        self._otel_context = otel_context
+
+    def unwrap(self):
+        """Returns the wrapped OpenTelemetry `SpanContext` object."""
+
+        return self._otel_context
+
+    @property
+    def baggage(self):
+        return {}
+        # TODO: Implement.
+
+
+class SpanWrapper(opentracing.Span):
+    def __init__(self, tracer, context, span: Span):
+        self._otel_span = span
+        opentracing.Span.__init__(self, tracer, context)
+
+    def unwrap(self):
+        """Returns the wrapped OpenTelemetry `Span` object."""
+
+        return self._otel_span
+
+    @property
+    def context(self):
+        return self._context
+
+    @property
+    def tracer(self):
+        return self._tracer
+
+    def set_operation_name(self, operation_name):
+        self._otel_span.update_name(operation_name)
+        return self
+
+    def finish(self, finish_time: float = None):
```

Comment:
Probably this should be `typing.optional[float]`, not sure why mypy doesn't comply.

related: https://github.com/open-telemetry/opentelemetry-python/pull/174#discussion_r331357011

Reply:
You are right, however I'm considering removing type hints completely from the bridge since we cannot use any OpenTracing types. See https://gitter.im/open-telemetry/opentelemetry-python?at=5d9b74cd7aa5602ffc86c16c

Reply:
I've removed typing information from the package since IMO it's better to omit typing altogether than to include *partial* typing info. We can revisit this at a later stage.